src/common/cache/src/todo

```diff
@@ -0,0 +1,12 @@
+- cache data crc
+- metrics
+- performance evaluation
+    - without threshold
+    - just put bytes to disk (no sync)
+    - what happens if cache is full? will performance degrade significantly?
+- if performance do not meet requirement
+    - consider make it tiered
+     - a in-memory moka at front
+     - disk cache layer listen to the eviction event (serving in dedicate thread pool)
+- misc: restart maybe slow if cached a large amount of data, consider make the startup of data cache layer async
```

Comment:
We can always keep the cache key in the memory, and the data on disk?

Reply:
cache keys (and their size) are kept in memory (an internal lru cache)

Reply:
> an in-memory moka at front

the worries here are if  huge tables are being full-table-scanned for the first time, putting all the new cache to disk may hurt query performance badly

Reply:
Ideally, the cache should probably not be placed entirely on disk as a result of a particular full table scan.

According to some papers in recent years, delaying the time for new data blocks to enter the cache appropriately may have performance benefits until more sophisticated methods are introduced.